openassessment/templates/openassessmentblock/student_training/student_training_error.html

```diff
@@ -6,7 +6,7 @@
         <div class="wrapper--step__content">
 
             <div class="step__message message message--incomplete">
-                <h3 class="message__title">{% trans "Error Loading Student Training Examples" %}</h3>
+                <h3 class="message__title">{% trans "Error Loading Learner Training Examples" %}</h3>
 
                 <div class="message__content">
                     <p>{% trans "We couldn't load the student training step of this assignment." %}</p>
```

Comment:
@dianakhuang In line with @andy-armstrong 's suggestion to remove the contraction in messages elsewhere, can we change this string to: "The Learner Training Step for this assignment could not be loaded." Thanks!